slave-maven/Dockerfile.rhel7

```diff
@@ -12,13 +12,13 @@ LABEL com.redhat.component="jenkins-slave-maven-rhel7-docker" \
       io.k8s.description="The jenkins slave maven image has the maven tools on top of the jenkins slave base image." \
       io.openshift.tags="openshift,jenkins,slave,maven"
 
-ENV MAVEN_VERSION=3.3 \
+ENV MAVEN_VERSION=3.5.0 \
```

Comment:
the rhel image is still stuck at 3.3 because there is no rhel scl packaging of 3.5 yet.